easybuild/easyconfigs/c/CGAL/CGAL-4.7-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,33 @@
+name = 'CGAL'
+version = '4.7'
+
+homepage = 'http://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient 
+ and reliable geometric algorithms in the form of a C++ library."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'strict': True}
+
+# note: source URL needs to be updated for a new version (checksums too)!
+source_urls = ['https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-4.7/']
+sources = [SOURCE_TAR_XZ]
+checksums = ['623d91fb2ab0a35049dc6098a0f235cc']
+
+pythonversion = '2.7.10'
+versionsuffix = "-Python-%s" % pythonversion
+
+dependencies = [
+    ('GMP', '6.0.0a'),
+    ('Boost', '1.59.0', versionsuffix),
+    ('MPFR', '3.1.3'),
+    ('Qt5', '5.5.1'),
+]
+
+configopts = "-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include\;$EBROOTLIBGLU/include "
+configopts += "-DOPENGL_gl_LIBRARY=$EBROOTMESA/lib/libGL.so -DOPENGL_glu_LIBRARY=$EBROOTLIBGLU/lib/libGLU.so "
```

Comment:
`$EBROOTMESA`, but no direct dependency on `Mesa`? same for libGLU

include them as explicit deps?